packages/instrumenter/src/transformers/babel-transformer.ts

```diff
@@ -156,15 +170,23 @@ export const transformBabel: AstTransformer<ScriptFormat> = (
    */
   function* mutate(node: NodePath): Iterable<Mutable> {
     for (const mutator of mutators) {
-      for (const replacement of mutator.mutate(node)) {
-        yield {
-          replacement,
-          mutatorName: mutator.name,
-          ignoreReason:
-            directiveBookkeeper.findIgnoreReason(node.node.loc!.start.line, mutator.name) ??
-            findExcludedMutatorIgnoreReason(mutator.name) ??
-            ignorerBookkeeper.currentIgnoreMessage,
-        };
+      if (options.runLevel === undefined || mutator.name in options.runLevel) {
+        let propertyValue = undefined;
+        if (options.runLevel !== undefined) {
+          const propertyName = mutator.name as keyof MutationLevel;
+          propertyValue = getPropertyByName(options.runLevel, propertyName);
+        }
```

Comment:
I think we can replace all of these simply with
`const propertyValue = options.runLevel?.[mutator.name as keyof MutationLevel] as string[];`

This will allow us to get rid of the getPropertyByName method too